pages/pipelines/buildkite_secrets.md

```diff
@@ -69,11 +69,10 @@ Buildkite secrets are designed, with the following controls in place:
 - All access to the secrets are logged.
 - Employee access to secrets is strictly limited and audited.
 
-> 📘 Buildkite secrets is not a zero-knowledge system
-> Buildkite owns, stores and manages the keys used for encrypting the secrets stored in the service at rest and in transit. Buildkite recommend customers implement additional controls to manage the lifecycle of secrets stored within Buildkite secrets, in addition to any monitoring capability you may require in line with your risk appetite.
-
 ## Best practices
 
+Buildkite secrets is not a zero-knowledge system, whereby Buildkite owns, stores, and manages the keys used for encrypting the secrets stored in the service at rest and in transit. You should implement additional controls to manage the lifecycle of secrets stored within Buildkite secrets, in addition to any monitoring capability you may require in line with your risk appetite. For example:
+
```

Comment:
FYI only @matthewborden , I hope you don't mind my moving the content from this note to the top of the **Best practices** section. It seemed that this content was better-suited to this section as it refers to the need to 'implement additional controls', which are effectively what are listed in this "Best practices" section.

While in the note, it felt as though the reader was left up in the air as to what these additional controls wer.